ci/rhel7/testing/saltstack-integration

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '600', daysToKeepStr: '7')),
+    disableConcurrentBuilds(),
+])
+
+pipeline {
+  environment { 
+      VERSION = "rhel7"
+      FLAVOR = "products-testing"
```

Comment:
Here I see a little naming problem that might cause confusion. All pipelines that uses `FLAVOR="products-testing"` should always be referred as "products-testing" (and not only to "testing" since this is a different package meant for openSUSE). More info about the Salt OBS packages: https://github.com/openSUSE/salt/wiki#projects

Therefore I would change the path and files to ensure all "products-testing" flavored pipelines are stored under `ci/*/products-testing/*`. In case we want to add pipeline for "testing" flavor, then new files should be added here.

Reply:
:+1: